packages/modules/ad-personalization/src/content/StrategyFactory.ts

```diff
@@ -0,0 +1,21 @@
+import { BaseStrategy } from './BaseStrategy';
+import * as Strategies from './strategies/Strategies';
```

Comment:
When file will be names 'index.js' then: import * as Strategies from './strategies';